cmake/sdlchecks.cmake

```diff
@@ -608,6 +608,10 @@ macro(CheckWayland)
         if(PKG_LIBDECOR_FOUND)
             set(HAVE_WAYLAND_LIBDECOR TRUE)
             set(HAVE_LIBDECOR_H 1)
+            if(PKG_LIBDECOR_VERSION VERSION_GREATER_EQUAL 0.1.1)
```

Comment:
Instead of hardcoding the libdecover version, we can also use [`cmake_check_function`](https://cmake.org/cmake/help/latest/module/CheckFunctionExists.html).

```cmake
# Add this to the section in the top CMakeLists.txt where we include various modules
include(CheckFunctionExists)

# Add this here
cmake_push_check_state()
set(CMAKE_CHECK_REQUIRED_FLAGS "${CMAKE_CHECK_REQUIRED_FLAGS} ${PKG_LIBDECOR_CFLAGS}")
set(CMAKE_CHECK_REQUIRED_LIBRARIES "${CMAKE_CHECK_REQUIRED_FLAGS} ${PKG_LIBDECOR_LDFLAGS}")
checK_function_exists(libdecor_frame_get_max_content_size HAVE_LIBDECOR_FRAME_GET_MAX_CONTENT_SIZE)
checK_function_exists(libdecor_frame_get_min_content_size HAVE_LIBDECOR_FRAME_GET_MIN_CONTENT_SIZE)
cmake_pop_check_state()
```